src/Materials/Textures/babylon.renderTargetTexture.ts

```diff
@@ -49,7 +49,7 @@
         */
         public onAfterUnbindObservable = new Observable<RenderTargetTexture>();
 
-        private _onAfterUnbindObserver: Observer<RenderTargetTexture>;
+        protected _onAfterUnbindObserver: Observer<RenderTargetTexture>;

```

Comment:
My implementation of BABYLON.MultiRenderTargetTexture is a child class of BABYLON.RenderTargetTexture, thus the change to protected for some properties.